### DISCUSSION TITLE
Theme templates?

### DISCUSSION THREAD

**Discussion by Paradoxum** [mathoudebine/turing-smart-screen-python#375] [Themes]
Can someone knock up some basic templates for us graphically challenged people?

Just with visual borders and lines and such for the various heading and sensor readings and we can modify the files  / change background etc,

Thanks!

**Comment by mathoudebine** [comment]
Hi, I am not much of a designer myself so I can't help you with this, but you may find some templates in this thread here: https://github.com/mathoudebine/turing-smart-screen-python/discussions/218 some of them may not have been made into a theme yet
